### ISSUE TITLE
Noosfero

### ISSUE THREAD

**Issue by desci** [piratas/gti#15]
Quem tem o backup do noosfero?

Tem documentos do partido que só tem lá e tem gente pedindo o histórico das discussões e conteúdo de lá.

**Comment by aurium** [comment]
@gomex deve saber quem tem.

**Comment by gomex** [comment]
O que eu tinha, era antigo e estava em um HD externo que queimou :( Esse backup era feito automaticamente para o servidor suecia1 ou 2, não lembro de verdade :)

**Comment by tjackgnu** [comment]
@desci *provavelmente* alguém que está/foi na [Coolive](http://colivre.coop.br/noosfero/) ↔ [Noosfero](http://noosfero.org/) ↔ [PIRATAS](http://partidopirata.org/)...

**Comment by gomex** [comment]
@tjackgnu não, seria eu mesmo, ou alguém do GTI, a galera da Coolivre dava ajuda, não fazia intervenção física na infra :)

**Comment by desci** [comment]
@gomex Não achei nos backups do suecia1 e do suecia2 de outubro de 2012.

**Comment by desci** [comment]
Aparentemente os backups do noosfero estão no servidor da suecia. Estou salvando tudo localmente aqui, espero que o @gomex e o @guilhermebr também.

**Comment by desci** [comment]
Estão salvos no **entwickler** e em três discos criptografados distribuídos em três casas diferentes em Porto Alegre.

**Comment by iannsp** [comment]
Existem planos de tornar publico os dados do noosfero?

**Comment by desci** [comment]
Comecei a ler o [manual de instalação](https://gitlab.com/noosfero/noosfero/blob/master/INSTALL.md).

@gomex com o docker tem como dar shell pras pessoas poderem ajudar a botar no ar o noosfero lá no **entwickler**?

**Comment by desci** [comment]
Se não tiver sistema de arquivos root isolado com shell eu uso o LXC que já ta instalado no servidor

**Comment by desci** [comment]
Proposta: centralizar nesta issue a discussão acerca de forum / rede social do partido, unir as outras duas issues #26 e #33 nesta, documentar toda a histórica discussão em relação a isto aqui.

**Comment by desci** [comment]
Adicionando comentário de #33:

> Tarefa relacionada: documentar a história do Noosfero, reunir links e discussões acerca de redes socias que já tivemos, a construção com o gnu social, a experiência com o discourse, com o loomio, com o telegram.

**Comment by desci** [comment]
Cheguei a propor usar blockchain como rede social em #34

**Comment by desci** [comment]
Registrando que chegaram a propor (novamente) phpBB no grupo de telegram, eu criei um container do docker pra uma das pessoas que propôs isto testar lá. Não há resultado registrado.

Botei no ar em carater de teste GNU Social: <https://social.pirata.xyz>.

Não foi mencionado neste tópico, mas existe um Discourse no ar em <http://partidopiratamg.org>.

O uso do Telegram como ferramenta pra tirar as pessoas do feisse continua intenso, sendo que isto teve diversos desdobramentos.

No vácuo da inércia, os mesmos de sempre continuam usando o loomio como blog e forum.

**Comment by desci** [comment]
Agregando: <https://pirata.hackpad.com/Criao-do-Frum-Pirata-EIxx9LIDa57>

---

# Criação do Fórum Pirata

## 1. Apresentação:

A aproximação do partido pirata da sociedade é um dos nossos desafios. O uso de fóruns na internet para se discutir tecnologia e cultura Geek é algo bem comum. A Criação de um fórum "pirata" irá trazer as pessoas para discutirem diversos assuntos da sociedade dentro do "território' pirata, aproximando indiretamente o partido da comunidade geek, aficionados por tecnologia, e outros.  
Exemplos de fóruns de tecnologia, onde também é discutida política:
<http://www.hardmob.com.br/forum.php>
<http://adrenaline.uol.com.br/forum>
O assunto principal nestes fóruns é tecnologia, mas existem tópicos sobre política e subsidiariamente esse tema é discutido nestes espaços

## 2. Objetivo Geral:
- Criar um fórum que possa ser usado por Associados e não associados
- Criar níveis de conteúdo (usuário anônimo/usuário registrado/pirata associado)
- Discutir primariamente Tecnologia (telecomunicações, Hardware, software, Gaggets, consoles, etc)
- Discutir de forma secundária questões política
- Discutir o partido pirata (somente associados)

## 3. Plataforma:
Phpbb (https://www.phpbb.com/)

### 3.1. Requisitos do server:

Php e SQL

## 4. Estrutura do Fórum:

Tópicos/Subtópicos:

- Tecnologia e Telecomunicações

  - Telefonia Móvel / Telefonia Fixa

  - Internet Móvel

  - Internet FIxa

  - TV por assinatura

- Eletrônicos

  - Câmeras digitais

  - Audio

  - Aparelhos de TV

- Hardware

  - Notebooks

  - Processadores

  - Placa mãe

  - Placa de Video

  - Periféricos

- Software

  - Código Fechado (windows, adobe, etc)

  - Código Livre (Linux e GNU)

- Games

  - Mobile

  - PC

  - Consoles

- Política

  - Saúde

  - Educação

  - Estado

  - Cultura

  - etc

- Partido Pirata (Restrito a associados)

**Comment by desci** [comment]
Registrando que mais de uma vez foi sugerido o uso do [reddit](https://github.com/reddit/reddit).

**Comment by tjackgnu** [comment]
@desci eu uso o [Vagrant](https://www.vagrantup.com/) para deixar rodando a porcaria da [ZeroNet](https://zeronet.io/) sem precisar ler mais do que as 4 linhas da receita de bolo do _"rautu"_; foi o jeito mais fácil de evitar as incomodações que eu mesmo criava aqui no _armário_... _JFTR_ ⚜

**Comment by desci** [comment]
Social Pirata agora está em http://social5dgegf5a7k.onion/

O domínio pirata.xyz não foi renovado: https://wiki.partidopirata.org/blog/Torify/
